test/form-builder/components/FormDetailContainer.spec.js

```diff
@@ -182,6 +183,21 @@ describe('FormDetailContainer', () => {
     sinon.assert.calledWith(httpInterceptor.get, formResourceURL);
   });
 
+  /* it('should call FormDetail with  appropriate controls', () => {
+    sinon.stub(httpInterceptor, 'get').callsFake(() => Promise.resolve(formData));
+    const dispatch = sinon.spy();
+    const wrapper = shallow(
```

Comment:
If this is not required, please remove it.

Reply:
done